harddrive-install-tree-relative.sh

```diff
@@ -17,7 +17,9 @@
 #
 # Red Hat Author(s): Jiri Konecny <jkonecny@redhat.com>
 
-TESTTYPE=${TESTTYPE:-"packaging"}
+# FIXME: fails on RHEL 8:
+# SourceSetupError: Nothing useful found for Hard drive ISO source at partition=/dev/sdb directory=/repo/
```

Comment:
@jkonecny12 might be interested in this failure.

Reply:
Making note.